packages/tinacms-authjs/package.json

```diff
@@ -27,11 +27,11 @@
     "next": "12.3.4",
     "next-auth": "4.22.1",
     "react": "^18",
-    "react-dom": "^18",
-    "tinacms": "workspace:*"
+    "react-dom": "^18"
   },
   "devDependencies": {
     "@tinacms/scripts": "workspace:*",
+    "tinacms": "workspace:*",
```

Comment:
why is this now in devDependencies - we are still using AbstractAuthProvider form 'tinacms' in src/tinacms?

Reply:
@kldavis4 tinacms really a peer dependency. I  added it as a devDependencies so that we can get the correct types in the mono repo.  

It is assumed that anyone using `tinacms-authjs` will have tinacms installed. If if leave it as a dependency, the user could get two versions of tinacms.  (The version from `tinacms-authjs` and there own version)

Reply:
ok